src/EnergyPlus/HVACMultiSpeedHeatPump.cc

```diff
@@ -2836,53 +2748,44 @@ namespace HVACMultiSpeedHeatPump {
         // SUBROUTINE INFORMATION:
         //       AUTHOR:          Lixing Gu, FSEC
         //       DATE WRITTEN:    June 2007
-        //       MODIFIED         na
-        //       RE-ENGINEERED    na
 
         // PURPOSE OF THIS SUBROUTINE:
         // This subroutine is for sizing multispeed heat pump airflow rates and flow fraction.
 
-        // Using/Aliasing
-        using namespace DataSizing;
-
-        using PlantUtilities::RegisterPlantCompDesignFlow;
-
         // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
         int NumOfSpeedCooling; // Number of speeds for cooling
         int NumOfSpeedHeating; // Number of speeds for heating
         int i;                 // Index to speed
 
-        auto &MSHeatPump = state.dataHVACMultiSpdHP->MSHeatPump;
+        auto &MSHeatPump = state.dataHVACMultiSpdHP->MSHeatPump(MSHeatPumpNum);
```

Comment:
[src/EnergyPlus/HVACMultiSpeedHeatPump.cc:2855]:(style),[constVariable],Variable 'MSHeatPump' can be declared as reference to const